homeassistant/components/tradfri/config_flow.py

```diff
@@ -197,7 +198,7 @@ async def get_gateway_info(
 
         api = factory.request
         gateway = Gateway()
-        gateway_info_result = await api(gateway.get_gateway_info())
+        gateway_info_result = await api(gateway.get_gateway_info(), timeout=TIMEOUT_API)
 
         await factory.shutdown()
     except (OSError, RequestError) as err:
```

Comment:
Do we need to catch the timeout error here?

Reply:
Done.